include/cantera/thermo/Elements.h

```diff
@@ -84,6 +84,50 @@ namespace Cantera
 //! stable state at 298.15 K and 1 bar.
 #define ENTROPY298_UNKNOWN -123456789.
 
+//! Get a vector of the atomic symbols of the elements defined in Cantera.
+//! @since New in version 3.0
+const std::vector<std::string>& elementSymbols();
+
+//! Get a vector of the names of the elements defined in Cantera.
+//! @since New in version 3.0
+const std::vector<std::string>& elementNames();
+
+//! Get a map with the element symbols as keys and weights as values.
+/*!
+ * This is a constant in the application so it is only generated once
+ * when it is first needed.
+ *
+ * @since New in version 3.0
+ */
+const std::map<std::string, double>& elementSymbolToWeight();
+
+//! Get a map with the element names as keys and weights as values.
+/*!
+ * This is a constant in the application so it is only generated once
+ * when it is first needed.
+ *
+ * @since New in version 3.0
+ */
+const std::map<std::string, double>& elementNameToWeight();
+
+//! Get a map with the isotope symbols as keys and weights as values.
+/*!
+ * This is a constant in the application so it is only generated once
+ * when it is first needed.
+ *
+ * @since New in version 3.0
+ */
+const std::map<std::string, double>& isotopeSymbolToWeight();
```

Comment:
Or you could go even a step further and include the isotope weights (by name and symbol) in the `elementWeights` map, since there are no overlaps, which would reduce the number of API functions that need to exist and simplify searching when you want to allow either elements or isotopes (which I think covers all the cases we have).